docs/en/Getting-Started.md

```diff
@@ -1,81 +1,365 @@
+command to list the files from the S3 mounted directory.
+
+```bash
+$ ./bin/alluxio fs ls /mnt/s3
+87.86KB   06-20-2016 12:50:51:660  Not In Memory  /mnt/s3/sample_tweets_100k.csv
+933.21KB  06-20-2016 12:50:53:633  Not In Memory  /mnt/s3/sample_tweets_1m.csv
+149.77MB  06-20-2016 12:50:55:473  Not In Memory  /mnt/s3/sample_tweets_150m.csv
+9.61MB    06-20-2016 12:50:55:821  Not In Memory  /mnt/s3/sample_tweets_10m.csv
+```
+
+We can see the [newly mounted files and directories in the Alluxio web UI](http://localhost:19999/browse?path=%2Fmnt%2Fs3) as well. 
+
+With Alluxio's unified namespace, you can easily interact with data from different storage systems
+seamlessly. For example, with the `ls` shell command, you can recursively list all the files that
+exist under a directory.
+
+```bash
+$ ./bin/alluxio fs ls -R /
+26.22KB   06-20-2016 11:30:04:415  In Memory      /LICENSE
+1.00B     06-20-2016 12:28:39:176                 /mnt
+4.00B     06-20-2016 12:30:41:986                 /mnt/s3
+87.86KB   06-20-2016 12:50:51:660  Not In Memory  /mnt/s3/sample_tweets_100k.csv
+933.21KB  06-20-2016 12:50:53:633  Not In Memory  /mnt/s3/sample_tweets_1m.csv
+149.77MB  06-20-2016 12:50:55:473  Not In Memory  /mnt/s3/sample_tweets_150m.csv
+9.61MB    06-20-2016 12:50:55:821  Not In Memory  /mnt/s3/sample_tweets_10m.csv
+```
+
+This shows all the files under the root of the Alluxio file system, from all of the mounted storage
+systems. The `/LICENSE` file is in your local file system, while the files under `/mnt/s3/` are in
+S3.
+
+## [AWS Bonus] Accelerating Data Access with Alluxio
+
+Since Alluxio leverages memory to store data, it can accelerate access to data. First, let’s take a
+look at the status of a file in Alluxio (mounted from S3).
+
+```bash
+$ ./bin/alluxio fs ls /mnt/s3/sample_tweets_150m.csv
+149.77MB  06-20-2016 12:50:55:473  Not In Memory  /mnt/s3/sample_tweets_150m.csv
+```
+
+The output shows that the file is **Not In Memory**. This file is a sample of tweets. Let’s see how
+many tweets mention the word "kitten". With the following command, we can count the number of
+tweets with "kitten".
+
+```bash
+$ time ./bin/alluxio fs cat /mnt/s3/sample_tweets_150m.csv | grep -c kitten
+889
+
+real	0m22.857s
+user	0m7.557s
+sys	0m1.181s
+```
+
+Depending on your network connection, the operation may take over 20 seconds. If reading this file
+takes too long, you may use a smaller dataset. The other files in the directory are smaller subsets
+of this file.
+
+Now, let’s see how many tweets mention the word "puppy".
+
+```bash
+$ time ./bin/alluxio fs cat /mnt/s3/sample_tweets_150m.csv | grep -c puppy
+1553
+
+real	0m25.998s
+user	0m6.828s
+sys	0m1.048s
+```
+
+As you can see, it takes a lot of time to access the data for each command. Alluxio can accelerate
+access to this data by using memory to store the data. However, the `cat` shell command prevents
```

Comment:
it's not that it prevents the caching, it's just that it doesn't do the caching.